node/resolvers/Routes/index.ts

```diff
@@ -224,9 +224,14 @@ export default {
             }
           })
 
-        if (!settings.paymentTerms && getOrganizationById?.paymentTerms) {
-          settings.paymentTerms = getOrganizationById.paymentTerms
-        }
+          if (settings.paymentTerms && getOrganizationById?.paymentTerms) {
+            const intersection = settings.paymentTerms.filter((obj: any) => getOrganizationById.paymentTerms.some((obj2: any) => obj.id === obj2.id && obj.name === obj2.name));
```

Comment:
could we change `obj` and `obj2` to `ccPaymentTerms` and `orgPaymentTerms`, or something like that? and maybe add a comment here explaining why we are doing this?